kafka-proxy.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: kafka-proxy
+  version: 0.3.9
+  epoch: 0
+  description: Allows a service to connect to Kafka brokers without having to deal with SASL/PLAIN authentication and SSL certificates
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/grepplabs/kafka-proxy
+      tag: v${{package.version}}
+      expected-commit: 1839a885e16e05fe670033690999abf0d34f695a
+  
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@1.33.0
+  - uses: go/build
+    with:
+      packages: .
+      output: kafka-proxy
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: grepplabs/kafka-proxy
+    strip-prefix: v
```

Comment:
Can  you please add basic test for the package

Reply:
Yes!